### PR TITLE
Policy support functions

### PR DESCRIPTION
**Pull request by CDimonaco**
# Description

Policy support functions to build bodyguard policies

## How was this tested?

Automated tests already present

### CODE REVIEW

File: lib/trento/support/policy_helper.ex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+defmodule Trento.Support.PolicyHelper do
+  @moduledoc """
+
```
Comment (arbulu89) [inline lib/trento/support/policy_helper.ex:3]: I would remove this empty new line

File: lib/trento/support/policy_helper.ex [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  Helper functions for bodyguard policies
+  """
+  alias Trento.Users.User
+
+  def user_has_ability?(%User{abilities: abilities}, %{name: name, resource: resource}),
```
Comment (arbulu89) [inline lib/trento/support/policy_helper.ex:8]: Could we have some basic UT specifically for this?

**Review by arbulu89 — APPROVED** [review]
Looks good!
Could we have some test? :stuck_out_tongue:

File: lib/trento/support/policy_helper.ex (hunk 1 above)
Comment (CDimonaco) [reply lib/trento/support/policy_helper.ex]: done!

File: lib/trento/support/policy_helper.ex (hunk 2 above)
Comment (CDimonaco) [reply lib/trento/support/policy_helper.ex:7]: done :P it was already tested but it's good to have dedicated tests

Comment (arbulu89) [reply lib/trento/support/policy_helper.ex:7]: thank you!

**Pull request merged**